Admin/Configuration/ActionConfiguration.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace LAG\AdminBundle\Admin\Configuration;
+
+
+class ActionConfiguration
+{
+    protected $route;
+
+    protected $parameters;
+
+    protected $loadMethod;
+
+    protected $exports;
+
+    protected $order;
+
+    protected $target;
+
+    protected $icon;
+
+    protected $batch;
+
+    public function __construct(array $configuration)
+    {
+        $this->loadMethod = $configuration['load_method'];
+        $this->route = $configuration['route'];
+        $this->parameters = $configuration['parameters'];
+        $this->exports = $configuration['export'];
+        $this->order = $configuration['order'];
+        $this->target = $configuration['target'];
+        $this->icon = $configuration['icon'];
+        $this->batch = $configuration['batch'];
+    }
+
+    /**
+     * @return mixed
+     */
+    public function getRoute()
```

Comment:
the route Symfony class is use in RoutingLoader, but it will have no added value.